File: src/main/java/tw/dworker/Hello.java
package tw.dworker;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new Person().hello();
//        person.hello();
        Person person = new Person("muscle",66.5f,1.7f);

        System.out.println();
        System.out.println(person.name+"的\n"+"BMI為"+person.bmi());

        int score = 88;
        System.out.println(score > 60 && score < 90);
        System.out.println(score < 60 || score > 90);

        char c = 'A';
        System.out.println(c < 'a');


       /* int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        String name = "muscle";*/


    }
}

File: src/main/kotlin/tw/dworker/Hello.kt
package tw.dworker

fun main() {
//    print("Hello World")
//    Human().hello()
    val h = Human(66.5f,1.7f,"muscle")
    println(h.bmi())

    val score = 88
    println( score > 60)

    val c: Char = 'A'
    println(c.toInt() > 60)

    var a = "abcde"
    println(a.length)
    println(a.get(3))
    println(a.substring(3))

    var b : String? = "abcde"
    b = null
    println(b?.get(2))
    println(b?.substring(3))

/*    h.hello()
    h.hi()*/

/*    var age = 19
    age = 20
    var weight = 66.5f
    var name : String
    name = "muscle"*/


}

class Human(var weight : Float, var  height : Float, var  name: String = ""){
    init {
        println("$name"+"的"+"\n"+"體重為$weight" + "\n" + "身高為$height")
    }
    //次要建構子
//    constructor(name: String, weight: Float, height: Float):this(weight,height)

    fun bmi() : Float{
        val bmi = weight / ( height * height)
        return bmi
    }

    fun hello(){
        println("Hello World by kotlin...")
    }
    fun hi(){
        println("this is Hi..")
    }
}